sa/_db/migrations/20151008132352_MoveOCSPResponseToCertificateStatus.sql

```diff
@@ -0,0 +1,23 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+ALTER TABLE `certificateStatus` ADD COLUMN (`ocspResponse` mediumblob);
+UPDATE certificateStatus,ocspResponses SET certificateStatus.ocspResponse = ocspResponses.Response;
+DROP TABLE `ocspResponses`;
```

Comment:
(Even if we're doing full outages for the deploy, we'd want the change to revert the deploy and be able to continue using that table.)